pkgs/development/compilers/ponyc/default.nix

```diff
@@ -111,5 +111,9 @@ stdenv.mkDerivation (rec {
     license = licenses.bsd2;
     maintainers = with maintainers; [ kamilchm patternspandemic redvers ];
     platforms = [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" "aarch64-darwin" ];
+    # Broken for Musl at 2024-01-17. Tracking issue:
+    # https://github.com/NixOS/nixpkgs/issues/281592
+    # Error: pop_var_context: head of shell_variables not a function context
```

Comment:
```suggestion
```
that is just a follow up error in bash and happens often when the build fails.

Reply:
Yeah, I didn't know back then. I know now though.

Pony is broken in master currently.